javascript/packages/linter/src/rules/html-anchor-require-href.ts

```diff
@@ -0,0 +1,41 @@
+import { BaseRuleVisitor, getTagName, hasAttribute } from "./rule-utils.js"
+
+import { Rule, LintMessage } from "../types.js"
+import type { HTMLOpenTagNode, Node } from "@herb-tools/core"
+
+class AnchorRechireHrefVisitor extends BaseRuleVisitor {
+  visitHTMLOpenTagNode(node: HTMLOpenTagNode): void {
+    this.checkATag(node)
+    super.visitHTMLOpenTagNode(node)
+  }
+
+  private checkATag(node: HTMLOpenTagNode): void {
+    const tagName = getTagName(node)
+
+    if (tagName !== "a") {
+      return
+    }
+
+    if (!hasAttribute(node, "href")) {
+      this.addMessage(
+        "Missing required `href` attribute on `<a>` tag.",
```

Comment:
How do you feel about making the message more actionable?
```suggestion
        "Add an `href` attribute to `<a>` to ensure it is focusable and accessible.",
```

Reply:
Very good idea 👍